docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -868,87 +859,6 @@ GitLab Repository Access
            continuous-integration:
                vcs-credentials: the.id.of.the.username.and.password.credentials.from.jenkins
 
-GitLab to Jenkins push notification token
```

Comment:
Note for other reviewers: For quite some time now Artemis sends the notification directly to Jenkins instead of relying on a Webhook from GitLab to Jenkins. Therefore, this token is no long required.